airflow/ui/src/pages/DagsList/DagsList.tsx

```diff
@@ -254,7 +260,11 @@ export const DagsList = () => {
         <DataTable
           cardDef={cardDef}
           columns={columns}
-          data={data.dags}
+          // We need to rename latest_dag_runs so that react-table can handle the sort correctly
+          data={data.dags.map((dag) => ({
+            ...dag,
+            last_run_start_date: dag.latest_dag_runs,
```

Comment:
This feels wrong? Isn't `latest_dag_runs` an array?

Reply:
Updated with an empty field vs overwriting it.